src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java

```diff
@@ -114,6 +121,7 @@ protected String getHeightResolution() {
 
     /**
      * Gets the selected frame for video converter.
+     *
```

Comment:
It should be "@return a String....."

Reply:
it's already solved dani could you check it